packages/qwik-city/buildtime/build.ts

```diff
@@ -46,6 +47,70 @@ export async function updateBuildContext(ctx: BuildContext) {
   return ctx.activeBuild;
 }
 
+function rewriteRoutes(ctx: BuildContext, resolved: ReturnType<typeof resolveSourceFiles>) {
+  if (ctx.opts.rewriteRoutes) {
+    for (let rewriteIndex = 0; rewriteIndex < ctx.opts.rewriteRoutes.length; rewriteIndex++) {
```

Comment:
I prefer forEach because it's more readable, anyway it looks good to me.

Reply:
I can change it if you prefer